src/include/tdb_defs.h

```diff
@@ -36,6 +36,8 @@
 #include <string>
 #include <tiledb/tiledb>
 
+constexpr size_t READ_BATCH_SIZE = 100000000;
```

Comment:
Lets make this a configuration option for the user at runtime. Let's also be clear what this value is (bytes?).

Reply:
Made this configurable and added more clarification about the value. This is expressed in number of array cells.